### DISCUSSION TITLE
Approval of account

### DISCUSSION THREAD

**Discussion by raiyanyahya** [community/community#50033] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Team,

Really awesome to sponsor and get sponsors for open source efforts.

I wanted to activate GitHub Sponsors on my account but it remains pending. It has been close to a week now. Is it common that the approval takes so much time?

Thank you all!

Regards.

**Comment by ademakdogan** [comment]
Hello! Congratulations on wanting to activate GitHub Sponsors on your account. The approval process can sometimes take a few days to a week or more, depending on the volume of requests and the verification process that GitHub goes through to ensure the legitimacy of the account. However, if it has been more than a week, you may want to reach out to GitHub support to inquire about the status of your application. They should be able to provide more information and assist you in resolving any issues that may be causing the delay. Good luck!

**Comment by jricciardi** [comment, marked as answer]
👋 Hey there @raiyanyahya! It looks like your account is now Sponsorable. Check out your profile under your picture for the button or head to https://github.com/sponsors/raiyanyahya 😄
